MonitorControl.xcodeproj/project.pbxproj

```diff
@@ -311,9 +311,6 @@
 			isa = PBXNativeTarget;
 			buildConfigurationList = 56754EB81D9A4016007BCDC5 /* Build configuration list for PBXNativeTarget "MonitorControl" */;
 			buildPhases = (
-				28D1DE0C227FCFAF004CB494 /* [Format] Run SwiftFormat */,
-				F03A8DF01FFB9D4C0034DC27 /* [Lint] Run SwiftLint */,
-				6CB1BDB2253C7EBE00B52124 /* [Localization] Run BartyCrouch */,
```

Comment:
I'd personally like to keep these scripts, since it keeps the codebase tidy and translation files up-to-date. I'm pretty sure these were probably removed to make it easier to build/test right?

Reply:
Yes, of course. I just removed them temporarily.